samples/net/sntp_client/prj.conf

```diff
@@ -0,0 +1,31 @@
+# General config
+CONFIG_NEWLIB_LIBC=y
+
+# Networking config
+CONFIG_NETWORKING=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_IPV6=y
+CONFIG_NET_UDP=y
+CONFIG_NET_SHELL=y
+
+# Network driver config
+CONFIG_TEST_RANDOM_GENERATOR=y
+
+# Network address config
+CONFIG_NET_APP_SETTINGS=y
+CONFIG_NET_APP_NEED_IPV4=y
+CONFIG_NET_APP_NEED_IPV6=y
+CONFIG_NET_APP_MY_IPV4_ADDR="192.0.2.1"
+CONFIG_NET_APP_PEER_IPV4_ADDR="192.0.2.2"
+CONFIG_NET_APP_MY_IPV6_ADDR="2001:db8::1"
+CONFIG_NET_APP_PEER_IPV6_ADDR="2001:db8::2"
+
+# Network debug config
+CONFIG_NET_LOG=y
+
+# qemu_x86 needs stack size larger than 1024
+CONFIG_MAIN_STACK_SIZE=1280
```

Comment:
I wonder why you would need to do this. I have not set this in much larger applications.

Reply:
Need to do this since both CONFIG_NET_IPV4 and CONFIG_NET_IPV6 are enabled. I will check it again.

Reply:
This setting might be necessary when debug message is enabled. I will remove it.